src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
+    }
+
+    /**
+     * @return string
+     */
+    public function getCurrentConversation()
+    {
+        if (is_null($this->currentConversation)) {
+            $cacheId = $this->getId() . '_current_conversation';
+            if (($currentConversation = $this->cache->fetch($cacheId)) !== false) {
+                $this->currentConversation = $currentConversation;
+
+                return $currentConversation;
+            };
+
+            $this->setCurrentConversation(HelloConversation::class);
```

Comment:
I think the first one makes sense. But what happens if i don't want to use the whole conversation system?

Also, If I ask a question to the user and I'm expecting an answer, how would you handle that in this conversation ? I'd like to see an example.

Example questions:
- What is your age? (Bot is expecting an integer value between 18-25).
- User responds: twenty years old
- Bot replies: Invalid Answer, Please tell me in numeric value.
- User responds: 17
- Bot replies: Sorry, Only people who're older than 18 can use this bot.
- User responds: 19
- Bot replies: Thank you.
- Bot sends next question: Where are you from?
...

Conversation continues.